misc/demo/assets/app.js

```diff
@@ -47,6 +47,19 @@ angular.module('ui.bootstrap.demo', ['ui.bootstrap', 'plunker', 'ngTouch', 'ngAn
 .controller('DownloadCtrl', DownloadCtrl);
 
 function MainCtrl($scope, $http, $document, $uibModal, orderByFilter) {
+  $scope.latestVersion = false;
+
+  // Grab old version docs
+  $http.get('versions-mapping.json')
+    .then(function(result) {
+      $scope.latestVersion = true;
```

Comment:
Does this imply the versions-mapping.json will only be in the root directory? Shouldn't the dropdown be available for the old docs too?